### PR TITLE
Improve CCloud CLI script

### PR DESCRIPTION
**Pull request by ybyzek**
Improvement by @rmoff

### CODE REVIEW

**Comment by rspurgeon** [comment]
@rmoff could you provide some context for this change please?

File: ccloud/beginner-cloud/start.sh [hunk 1]
```diff
@@ -109,16 +109,13 @@ if [[ $status != 0 ]]; then
 fi
 
 echo -e "\n# Produce 10 messages to topic $TOPIC1"
-echo '(for i in `seq 1 10`; do echo "${i}" ; done) | \'
-echo "timeout 10s ccloud kafka topic produce $TOPIC1"
-(for i in `seq 1 10`; do echo "${i}" ; done) | timeout 10s ccloud kafka topic produce $TOPIC1
```
Comment (rspurgeon) [inline ccloud/beginner-cloud/start.sh:114]: does the removal of the usage of `timeout` have a side effect that we need to consider?

Comment (rmoff) [reply ccloud/beginner-cloud/start.sh:114]: with the timeout in place the producer might exit before all messages are sent

**Comment by rmoff** [comment]
> @rmoff could you provide some context for this change please?

I was going through the script for recording, and figured `tee` would fit better here to actually echo what's being sent to the producer, instead of faking it by sending data to the producer and then echoing to screen the output of the same statement that was used to generate the data. 

It's not a biggie and probably not worth the time being spent looking at it, so feel free to close :)

File: ccloud/beginner-cloud/start.sh (hunk 1 above)
Comment (rspurgeon) [reply ccloud/beginner-cloud/start.sh:114]: Nice observation.  I'm wondering about the failure case, does the producer have it's own built in timeout to handle misconfigurations or timeouts for us so we don't just hang here indefinitely?

**Comment by rspurgeon** [comment]
@rmoff closed this for now, we just wanted to test out the side effect of removing the timeout entirely.  Tracking in DEVX-1780

**Pull request closed without merging**